utils/walker/git.go

```diff
@@ -172,10 +172,19 @@ func clonewalk(g *Git) error {
 	}
 
 	// If recurse mode is off, we only walk the root directory passed with g.root
-	files, err := ioutil.ReadDir(filepath.Join(path, g.root))
+	entries, err := os.ReadDir(filepath.Join(path, g.root))
 	if err != nil {
 		return err
 	}
+	files := make([]fs.FileInfo, 0, len(entries))
+	for _, entry := range entries {
+		file, err := entry.Info()
+		if err != nil {
+			return err
+		}
+		files = append(files, file)
+	}
+
```

Comment:
@pk-218 what's the need for this? Is this part of lint failures? 🤔

Reply:
The lint checks were failing here since `ioutil.ReadDir` is deprecated

As per the [documentation](https://pkg.go.dev/io/ioutil#ReadDir), the replacement is `os.ReadDir` but there is a change in the method return value. Hence, to accommodate the changes, I had to add this code.

Reply:
Ohh I see.